lib/irb/context.rb

```diff
@@ -649,6 +649,21 @@ def parse_command(code)
       end
     end
 
+    def colorize_code(input, complete:)
```

Comment:
I feel `colorize_input` express the intent better and is less likely to be confused with `Color.colorize_code`?

Reply:
It looks better, but I have one concern. colorize_input colorizes output. I think it is an input, but Reline's api calls it output.
Do you think the code below is acceptable? IMO... maybe yes?
```ruby
Reline.output_modifier_proc = proc do |output, complete:|
  IRB.CurrentContext.colorize_input(output, complete: complete)
end
```